feed-integration-webapp/src/main/java/org/bahmni/module/feedintegration/atomfeed/contract/patient/OpenMRSPerson.java

```diff
@@ -0,0 +1,167 @@
+package org.bahmni.module.feedintegration.atomfeed.contract.patient;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Date;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OpenMRSPerson {
+    private String uuid;
+    private String display;
+    private String gender;
+    private int age;
+    private Date birthdate;
+    private String birthtime;
```

Comment:
Why do we need all this data about a person in crater integration code? For creating a Customer, my guess is we only need minor details:  Name, Age, Gender, Patient ID, Address.

Other information here like birthtime, death details, preferredNames, etc... should not be needed. Can we not ignore those while constructing the OpenMRSPerson object? Else, more personal data is travelling across services.

Reply:
Irrelevant info has been removed.